.env.test

```diff
@@ -10,4 +10,6 @@ TRELLO_TOKEN='Test'
 TRELLO_BOARD='Test'
 TRELLO_TODO_LIST_ID='Test'
 TRELLO_DOING_LIST_ID='Test'
-TRELLO_DONE_LIST_ID='Test'
\ No newline at end of file
+TRELLO_DONE_LIST_ID='Test'
```

Comment:
These variables are no longer used, so we should remove them from here, and from the Travis setup